x-pack/plugins/monitoring/server/lib/metrics/classes/metric.ts

```diff
@@ -56,6 +57,7 @@ export class Metric {
   public usageField?: string;
   public periodsField?: string;
   public quotaField?: string;
+  public isMetricbeatOnly?: boolean;
```

Comment:
I'm tempted to change this to something like `isNotSupportedInInternalCollection` because I imagine these metrics will be available beyond Metricbeat at some point.

Reply:
Done!